.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: php
+
+dist: trusty
+sudo: false
+
+git:
+    depth: 1
+
+php:
+  - 7.1
+  - 7.2
+
+cache:
+    directories:
+        - $HOME/.composer/cache/files
+        - vendor
+        - .phpunit
+
+env:
+    global:
+        - SYMFONY_PHPUNIT_DIR=.phpunit
+
+install:
+    - composer --prefer-source install
```

Comment:
you should also add `./vendor/bin/simple-phpunit install` in the install section, so that the PHPUnit installation gets done in `install` rather than when running tests. This makes the build output less confusing.